airflow/serialization/__init__.py

```diff
@@ -15,5 +15,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
 """DAG serialization."""
+from __future__ import annotations
```

Comment:
This is an otherwise empty file -- we shouldn't add the import here I don't think? (Doesn't matter I guess)